app/src/main/java/com/google/developer/udacityalumni/fragment/ArticleFragment.java

```diff
@@ -99,6 +100,17 @@ public void onShareClick(String title) {
 //                ie: https://udacity-alumni-client.herokuapp.com/articles/medically-necessary-utilization-review-evidence-of-insurability
 //                TODO: Share DYNAMIC link to the article  (the 'url' above is a link to the article on the web app)
 //               Will implement later
+                try {
+                    Intent i = new Intent(Intent.ACTION_SEND);
+                    i.setType("text/plain");
+                    i.putExtra(Intent.EXTRA_SUBJECT, "Udacity Alumini");
+                    String sAux = "\nLet me recommend you this Article\n\n";
+                    sAux = sAux + url;
+                    i.putExtra(Intent.EXTRA_TEXT, sAux);
+                    startActivity(Intent.createChooser(i, "choose one"));
+                } catch(Exception e) {
+
+                }
```

Comment:
Looks good! Only suggestions I'd make would be to switch these hard coded `String`s for `xml` resources. I'd also remove the `try\catch` block.